website/source/docs/configuration/storage/google-cloud.html.md

```diff
@@ -34,11 +34,16 @@ storage "gcs" {
   account must have permission to read, write, and delete from the bucket. This
   can also be provided via the environment variable `GOOGLE_STORAGE_BUCKET`.
 
-- `credentials_file` `(string: <required>)` – Specifies the path on disk to a
+- `credentials_file` `(string: "<varies>")` – Specifies the path on disk to a
   Google Cloud Platform [service account][gcs-service-account] private key file
   in [JSON format][gcs-private-key]. This can also be provided via the
   environment variable `GOOGLE_APPLICATION_CREDENTIALS`.
 
+- `use_application_default_credentials` `(bool: false)` – Specifies to use GCP
+  [Application Default Credentials][adc]. This can only be used within Google
```

Comment:
I don't think it is quite accurate (or necessary) to say ADC can only be used within GCE, since a developer can use `gcloud auth application-default login` on their machine.

Reply:
I just don't think it's necessary to explain when ADC should be used-- might as well just point to Google's documentation since it could change over time. Also, I can't seem to find any documentation distinguishing where ADC should be used and where `GOOGLE_APPLICATION_CREDENTIALS` should be used apart from ADC.

Reply:
I can't point to any google doc that says it, but from my personal experience you would use `GOOGLE_APPLICATION_CREDENTIALS` in two places: you want to override the ADC, or they are not available where you are running.

Reply:
I see your point about not explaining when they should be used, change incoming